cob_bringup/robots/base_solo.launch

```diff
@@ -8,26 +8,26 @@
 	<include file="$(arg pkg_hardware_config)/common/upload_robot.launch" >
 		<arg name="robot" value="$(arg robot)" />
 	</include>
+    <include file="$(find cob_default_robot_config)/upload_param.launch" >
+		<arg name="robot" value="$(arg robot)" />
+	</include>
 
-	<!-- upload default configuration parameters -->
-	<include file="$(find cob_default_robot_config)/$(arg robot)/upload_param_$(arg robot).launch" />
-	
 	<!-- start robot_state_publisher -->
 	<node pkg="robot_state_publisher" type="state_publisher" name="robot_state_publisher"/>
 
-	<include file="$(find cob_script_server)/launch/script_server.launch" />
 
 	<!-- start base driver -->
-	<include file="$(find cob_bringup)/drivers/base_driver.launch" >
+	<include file="$(find cob_bringup)/drivers/canopen_402.launch" >
```

Comment:
now we do not have a base_solo for the old bases anymore....we should consider a second launch file...or a launch argument....